source/core/src/main/com/csse3200/game/entities/Entity.java

```diff
@@ -48,14 +49,17 @@ public enum EnemyType {
 
 
   public Entity() {
-
     id = nextId;
     nextId++;
 
     components = new IntMap<>(4);
     eventHandler = new EventHandler();
   }
 
+  public void setIsPlayer(boolean status) {this.isPlayer = status;}
```

Comment:
Maybe we should have an EntityType that includes EnemyType, Player, Friendly NPC type etc

Reply:
We could, but currently every single enemy has it's own enum value - that should be refactored first. For now I don't think that's relevant to this pr although that should be refactored later.